### PR TITLE
Job is done

### PR DESCRIPTION
**Pull request by Stan0934**
- [DEMO LINK](https://stan0934.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://stan0934.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  padding-top: 2px;
+  font-size: 14px;
+  height: 42px;
+  line-height: 40px;
+  padding-left: 33px;
+}
+
+.search__bar:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
```
Comment (TarasHoliuk) [inline src/style.css:45]: You've already set it (and `border-radius`) in `.search__bar`. No need to do it again

Comment (TarasHoliuk) [reply src/style.css:45]: the same for `focus`

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  font-weight: 400;
+}
+
+.search__bar:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
```
Comment (TarasHoliuk) [inline src/style.css:54]: `3px` is needed for tests?

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  font-weight: 900;
+}
+
+.form {
+  /* margin-left: 40px; */
```
Comment (TarasHoliuk) [inline src/style.css:59]: don't leave comments

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.small__form::before {
+  height: 11px;
+  width: 11px;
+  content: "";
+  background-image: url("images/Search.svg");
+  background-size: contain;
+  position: absolute;
+  background-repeat: no-repeat;
```
Comment (TarasHoliuk) [inline src/style.css:82]: these properties are the same for both - you can move them to `.form`

File: src/index.html [hunk 5]
```diff
@@ -10,7 +10,37 @@
...
+        type="text" 
+        placeholder="Try &QUOT;Los Angeles&QUOT;"
+        class="search__bar small"
+        name="small-search"
+        data-qa="small"
+        >
+      </form>
+    </div>
   </body>
+
```
Comment (TarasHoliuk) [inline src/index.html:45]: redundant line

File: src/index.html [hunk 6]
```diff
@@ -10,7 +10,37 @@
...
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="box">
+      <form 
+      class="form big__form" 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      >
```
Comment (TarasHoliuk) [inline src/index.html:20]:
```suggestion
      <form 
        class="form big__form" 
        action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
        method="post"
      >
```

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
pay attention to code style: [CHECKLIST](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

File: src/style.css (hunk 1 above)
Comment (Stan0934) [reply src/style.css:45]: +

File: src/style.css (hunk 3 above)
Comment (Stan0934) [reply src/style.css:59]: plus

File: src/style.css (hunk 4 above)
Comment (Stan0934) [reply src/style.css:82]: It's doesn't work. I don't know, how to make it work.

File: src/index.html (hunk 5 above)
Comment (Stan0934) [reply src/index.html:43]: plus

File: src/index.html [hunk 7]
```diff
@@ -10,7 +10,35 @@
...
+        placeholder="Try &QUOT;Los Angeles&QUOT;"
+        class="search__bar small"
+        name="small-search"
+        data-qa="small"
+      >
+    </form>
   </body>
-</html>
+  
+</html>
```
Comment (polosanya) [inline src/index.html:44]: New line in the end is required

**Review by polosanya — APPROVED** [review]
Good job!

**Pull request closed without merging**